elfpy/data/acquire_data.py

```diff
@@ -177,11 +177,22 @@ def main(
         # Query and add block_pool_info
         pool_info_dict = hyperdrive_interface.get_hyperdrive_pool_info(web3, hyperdrive_contract, block_number)
         # Set defaults
-        for key in db_schema.PoolInfo.__annotations__.keys():
-            if key not in pool_info_dict.keys():
+        for key in db_schema.PoolInfo.__annotations__:
```

Comment:
Can you add a TODO here to abstract this out? With the changes to hyperdrive_interface, it's on my list to pull the conversion between the interface to the db object into various functions